cantools/database/can/message.py

```diff
@@ -23,6 +32,17 @@
     from .formats.arxml import AutosarMessageSpecifics
     from .formats.dbc import DbcSpecifics
 
+LOGGER = logging.getLogger(__name__)
+
+# type aliases
```

Comment:
Comment suggestion: Type aliases. Introduced to reduce type annotation complexity while allowing for more complex encode/decode stategies like the one used for ARXML containers.

Reply:
done